src/app/layout.js

```diff
@@ -6,14 +6,22 @@ import Footer, { MinimalSocialsFooter } from "@/components/footer";
 const inter = Inter({ subsets: ["latin"] });
```

Comment:
If you dont want these font, remove this and its class. Declare everything from globals.css